### ISSUE TITLE
Add tools for generating mappings_XX.h

### ISSUE THREAD

**Issue by corona10** [python/cpython#84508]
BPO | [40328](https://bugs.python.org/issue40328)
--- | :---
Nosy | @hyeshik, @vstinner, @methane, @serhiy-storchaka, @corona10
PRs | <li>python/cpython#19602</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2020-04-19.13:13:59.400>
labels = ['3.9']
title = 'Add tools for generating mappings_XX.h'
updated_at = <Date 2020-05-25.04:14:49.537>
user = 'https://github.com/corona10'
```

bugs.python.org fields:
```python
activity = <Date 2020-05-25.04:14:49.537>
actor = 'corona10'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Demos and Tools']
creation = <Date 2020-04-19.13:13:59.400>
creator = 'corona10'
dependencies = []
files = []
hgrepos = []
issue_num = 40328
keywords = ['patch']
message_count = 4.0
messages = ['366777', '367682', '369844', '369845']
nosy_count = 5.0
nosy_names = ['hyeshik.chang', 'vstinner', 'methane', 'serhiy.storchaka', 'corona10']
pr_nums = ['19602']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue40328'
versions = ['Python 3.9']
```

</p></details>

**Comment by sorcio** [comment]
Hey @corona10, I happened upon this for other reasons. I found that Python's big5hkscs codec does not support the same character set that modern web browsers support.

There has been at least one version of HKSCS (2008) that updated the Big5-encoded character set, and one more recent version (2016) which only added characters to Unicode (with no Big5 mapping), but might adjust some of the existing mappings. I still need to understand if this might break compatibility.

Is it okay if I look into this and try to make a PR later? I've already done something for mappings_hk but need to look into the compatibility, so it might take a second.
